plugins/asfdata.py

```diff
@@ -321,6 +321,14 @@ def process_sequence(metadata, seq, sequence, load, debug):
         else:
             print(f'{seq} - split requires an existing sequence to split')
 
+    # count the number podlings
+    if 'size' in sequence:
+        if debug:
+            print(f'size: {sequence["size"]}')
+        print(seq)
+        if seq == 'podlings_size':
```

Comment:
Why only process size if the name is 'podlings_size'?
Also the debug and print statements apply regardless of the sequence name.
This all seems very fragile.

Reply:
Can you suggest any changes to improve this code? 
Remove print statement and what else?

Reply:
Why is it necessary to check the sequence name against 'podlings_size'?
Would it not be useful to allow size to be applied to other sequences?

An alternative approach might be to add support for adding a size attribute to sequences (and possibly other collections), in which case one could use sequence_name.size as a reference. This would be a lot more versatile.